examples/pipeline_wavernn/README.md

```diff
@@ -0,0 +1,36 @@
+This is an example pipeline for WaveRNN vocoder using the WaveRNN model trained on LJSpeech. WaveRNN model is based on the implementation from [this repository](https://github.com/fatchord/WaveRNN). The original implementation was
+introduced in "Efficient Neural Audio Synthesis". WaveRNN and LJSpeech are available in torchaudio.
```

Comment:
As a separate pull request, we could consider adding the plots for training and validation errors in this readme for the pre-trained weight. Either in #776 or another one.

Reply:
Good suggestion, I will have a separate pull request to add the plots.